dex/networks/btc/script.go

```diff
@@ -170,6 +170,15 @@ const (
 	// NOTE: witness data is not script.
 	RedeemP2WPKHInputWitnessWeight = 1 + 1 + DERSigLength + 1 + 33 // 109
 
+	// RedeemP2PKHInputTotalSize is the worst case size of a transaction
+	// input redeeming a P2WPKH output and the corresponding witness data.
+	// It is calculated as:
+	//
+	// 41 vbytes base tx input
+	// 109wu witness +  2wu segwit marker and flag = 28 vbytes
+	// total = 69 vbytes
+	RedeemP2PWKHInputTotalSize = RedeemP2WPKHInputSize + ((RedeemP2WPKHInputWitnessWeight + 2 + 3) / 4)
```

Comment:
OK that makes sense. So it it would be correct to have `MinimumTxOverhead = 10` and also have `MinimumSegwitTxOverhead = 11`, right? For example in `(w *spvWallet) sendWithSubtract`, the `unfundedTxSize` is slightly underestimated due to this. These are just tiny differences though and don't really effect anything.

Reply:
Sorta.  I think it's important to sum all the witness data first though, before dividing by 4 and rounding up.

Reply:
I mean it's certainly a safe estimate if it's better to err toward high estimates